Index: src/com/access/control/Country.java
===================================================================
diff --git a/src/com/access/control/Country.java b/src/com/access/control/Country.java
deleted file mode 100644
--- a/src/com/access/control/Country.java	(revision 3357fc26f5d553ae4e07c1d76e837213d2826ca1)
+++ /dev/null	(revision 3357fc26f5d553ae4e07c1d76e837213d2826ca1)
@@ -1,5 +0,0 @@
-package com.access.control;
-
-public class Country {
-
-}
Index: ../PracticePrograms/.idea/vcs.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/../PracticePrograms/.idea/vcs.xml b/../PracticePrograms/.idea/vcs.xml
new file mode 100644
--- /dev/null	(date 1742543700230)
+++ b/../PracticePrograms/.idea/vcs.xml	(date 1742543700230)
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="$PROJECT_DIR$/.." vcs="Git" />
+  </component>
+</project>
\ No newline at end of file
Index: src/com/xworkz/practice/Demo.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/com/xworkz/practice/Demo.java b/src/com/xworkz/practice/Demo.java
new file mode 100644
--- /dev/null	(date 1742788769481)
+++ b/src/com/xworkz/practice/Demo.java	(date 1742788769481)
@@ -0,0 +1,16 @@
+package com.xworkz.practice;
+
+public class Demo {
+    public int num1=10;
+    int num2=20;
+    private String name="Kantharaj";
+   public void display()
+    {
+        System.out.println(name);
+    }
+    private void number()
+    {
+        System.out.println(num1+""+num2);
+    }
+
+}
Index: src/com/xworkz/practice/DemoRun.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/com/xworkz/practice/DemoRun.java b/src/com/xworkz/practice/DemoRun.java
new file mode 100644
--- /dev/null	(date 1742788689420)
+++ b/src/com/xworkz/practice/DemoRun.java	(date 1742788689420)
@@ -0,0 +1,13 @@
+package com.xworkz.practice;
+
+public class DemoRun {
+    public static void main(String[] args) {
+        Demo demo=new Demo();
+        System.out.println(demo.num1);
+        System.out.println(demo.num2);
+        //System.out.println(demo.name);
+        demo.display();
+       // demo.number();
+    }
+}
+
Index: ../PracticePrograms/.idea/misc.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/../PracticePrograms/.idea/misc.xml b/../PracticePrograms/.idea/misc.xml
new file mode 100644
--- /dev/null	(date 1742543741036)
+++ b/../PracticePrograms/.idea/misc.xml	(date 1742543741036)
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectRootManager" version="2" default="true">
+    <output url="file://$PROJECT_DIR$/out" />
+  </component>
+</project>
\ No newline at end of file
Index: src/com/xworkz/practice/atcls/Demo2Run.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/com/xworkz/practice/atcls/Demo2Run.java b/src/com/xworkz/practice/atcls/Demo2Run.java
new file mode 100644
--- /dev/null	(date 1742788689304)
+++ b/src/com/xworkz/practice/atcls/Demo2Run.java	(date 1742788689304)
@@ -0,0 +1,15 @@
+package com.xworkz.practice.atcls;
+
+import com.xworkz.practice.Demo;
+
+public class Demo2Run {
+    public static void main(String[] args) {
+        Demo demo=new Demo();
+        System.out.println(demo.num1);
+        //System.out.println(demo.num2);
+        //System.out.println(demo.name);
+        demo.display();
+        demo.number();
+
+    }
+}
Index: ../PracticePrograms/.idea/.gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/../PracticePrograms/.idea/.gitignore b/../PracticePrograms/.idea/.gitignore
new file mode 100644
--- /dev/null	(date 1742543700189)
+++ b/../PracticePrograms/.idea/.gitignore	(date 1742543700189)
@@ -0,0 +1,3 @@
+# Default ignored files
+/shelf/
+/workspace.xml
